plugins/modules/mysql_replication.py

```diff
@@ -244,17 +246,63 @@
 executed_queries = []
 
 
+def uses_replica_terminology(cursor):
+    """Checks if REPLICA must be used instead of SLAVE"""
+    cursor.execute("SELECT VERSION() AS version")
+    result = cursor.fetchone()
+
+    if isinstance(result, dict):
+        version_str = result['version']
+    else:
+        version_str = result[0]
+
+    version = version_str.split('.')
+
+    ver_major = int(version[0])
+    ver_minor = int(version[1])
+    if '-' in version[2]:
+        ver_patch = int(version[2].split('-')[0])
+    else:
+        ver_patch = int(version[2])
```

Comment:
Why not use `LooseVersion` (`from distutils.version import LooseVersion`) instead of manual version parsing?

```.py
version = LooseVersion(version_str)
if 'mariadb' in version_str.lower():
    return LooseVersion('10.5.1') >= version
else:
    return LooseVersion('8.0.22') >= version
```

Reply:
@felixfontein definitely "why?" I'm asking myself:) Moreover i've used this function before many times:) Thanks for pointing out the obvious solution!